sass/_cta.scss

```diff
@@ -0,0 +1,13 @@
+.cta__wrapper {
+  @include component-box;
+  position: fixed;
+  top: 50%;
+  transform: translateY(-50%);
+  right: 0;
+  padding: 0;
+  z-index: 9999;
+  width: 3.75rem;
```

Comment:
Since width is applied to the child elements, this appears to work without having width here also, so you can remove this line (if it also works for you)